index.html

```diff
@@ -57,7 +57,7 @@
           <a href="https://twitter.com/ElijahHorton" target="_blank" rel="nofollow">@ElijahHorton</a></span>
         </div>
         <div class="heroku-version">
-          <a class="link-to-heroku" href="http://phpepl.herokuapp.com">Click for a PHP 5.5 version</a>
+          <a class="link-to-heroku" href="http://phpepl.herokuapp.com">Click for a PHP 5.6 version</a>
```

Comment:
Yeah, could be, but the http://phpepl.cloudcontrolled.com/ is a PHP 5.3 version, so 5.x as well...

Maybe, a good catch would be "Click for run it in the latest PHP version (in Heroku)". I guess Heroku will move to PHP 7 as soon as they can.